bind.go

```diff
@@ -159,6 +159,12 @@ func (b *DefaultBinder) bindData(destination interface{}, data map[string][]stri
 		for k, v := range data {
 			if isElemString {
 				val.SetMapIndex(reflect.ValueOf(k), reflect.ValueOf(v[0]))
+			} else if isElemInterface {
+				if len(v) == 1 {
```

Comment:
I am not certain that this behavior is not creating more problems.  Number of query params does not need to be fixed. You could have situations where user send request with multiple values and some time one. now at backend you are forced to real with situation that you could be dealing with  slice of strings of string.   This functionality could be some kind of search form with checkboxes etc.

Reply:
@aldas thanks for pointing this out. I agree having to deal with slice of strings and single string would be inconvenient. 

I made changes as per your suggestions. Now for interface type, it will produce always first string. Please review again.